pkg/persistence/disk_persistence_test.go

```diff
@@ -24,6 +26,9 @@ var (
 
 	pathToCurrent = fmt.Sprintf("%s/%s", dataDir, dirCurrent)
 	pathToArchive = fmt.Sprintf("%s/%s", dataDir, dirArchive)
+
+	errExpectedRead  = errors.New("cannot read from the storage directory: ")
+	errExpectedWrite = errors.New("cannot write to the storage directory: ")
```

Comment:
Why not `fmt.Errorf`?

Reply:
I think both are fine according to [go blog on errors](https://tip.golang.org/pkg/errors/#example_New). 
And in this case, I also do not have a format specifier